### DISCUSSION TITLE
I dont get the options in the video When I try to write the command vi

### DISCUSSION THREAD

**Discussion by omarghoneim3** [lnis-uofu/OpenFPGA#727] [Q&A]
I get never end loop after I write vi op
E325: ATTENTION
Found a swap file by the name ".op.swp"
          owned by: omar   dated: Thu Jul 21 21:40:19 2022
         file name: ~omar/OpenFPGA/op
          modified: YES
         user name: omar   host name: omar-VirtualBox
        process ID: 2779
While opening file "op"

(1) Another program may be editing the same file.  If this is the case,
    be careful not to end up with two different instances of the same
    file when making changes.  Quit, or continue with caution.
(2) An edit session for this file crashed.
    If this is the case, use ":recover" or "vim -r op"
    to recover the changes (see ":help recovery").
    If you did this already, delete the swap file ".op.swp"
    to avoid this message.

"op" [New File]

**Comment by tangxifan** [comment]
@omarghoneim3 Please check ``vi`` usage by using google.
